cmd/copyEnumeratorInit.go

```diff
@@ -497,13 +500,7 @@ func (cca *CookedCopyCmdArgs) createDstContainer(containerName string, dstWithSA
 		}
 		return err
 	case common.ELocation.BlobFS():
-		// TODO: Implement blobfs container creation
-		accountRoot, err := GetAccountRoot(dstWithSAS, cca.FromTo.To())
-		if err != nil {
-			return err
-		}
-
-		dsc := common.CreateDatalakeServiceClient(accountRoot, dstCredInfo, nil, options)
+		dsc, _ := sc.DatalakeServiceClient()
```

Comment:
error handling?

Reply:
They are available just above. It is just an overhead here.